pkg/snap/util/dqlite.go

```diff
@@ -131,3 +131,17 @@ func MaybeUpdateDqliteBindAddress(ctx context.Context, snap snap.Snap, hostPort
 	}
 	return nil
 }
+
+// RemoveNodeFromDqlite uses the Dqlite binary to remove a node from the Dqlite cluster.
+func RemoveNodeFromDqlite(ctx context.Context, snap snap.Snap, hostPort string) error {
```

Comment:
This endpoint is a bit dangerous. I'd prefer to have checks here to not remove the last leader, not break HA etc. but I know the effort for that is large.

Reply:
I completely agree. Maybe we can have separate cards for those? I think I won't be able to address those in this pulse. And since this whole feature does not have _that high_ of a priority, I guess we can afford to add these in the future pulses. WDYT?

Reply:
Indeed, let's create cards for those. If it's priority we can pull it in the current pulse.